### DISCUSSION TITLE
Flexdashboard in quarto

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#843] [Q&A] [answered]
It will be possible to work with flexdashboard in quarto in the near future?

**Comment by cderv** [comment, marked as answer]
A Quarto format equivalent to **flexdashboard** R package is something that we plan to do. As show in the Quarto FAQ : https://quarto.org/docs/faq/rmarkdown.html#i-use-x-bookdown-blogdown-etc..-what-is-the-quarto-equivalent, best estimate is Fall 2022 for now.

**Comment by ManuelSpinola** [comment]
Thank you very much.
